src/dialogs/rename.tsx

```diff
@@ -107,11 +137,12 @@ const RenameItemModal = ({ dialogResult, path, selected } : {
                   isInline
                 />}
                 <Form isHorizontal>
-                    <FormGroup fieldId="rename-item-input" label={_("New name")}>
+                    <FormGroup fieldId="rename-item-input" label={_("New filename")}>
                         <TextInput
                           value={name}
                           onChange={(_, val) => {
-                              setNameError(check_name(val, cwdInfo?.entries || {}));
+                              setNameError(checkName(val, cwdInfo?.entries || {}, selected));
+                              setOverrideFileName(checkCanOverride(val, cwdInfo?.entries || {}, selected));
```

Comment:
These 2 added lines are not executed by any test.

Reply:
what